requirements.txt

```diff
@@ -13,7 +13,7 @@ types-PyYAML==6.0.12.10
 types-dataclasses==0.6.6
 pytest==7.4.0
 trio==0.22.1
-anyio@git+https://github.com/agronholm/anyio.git
+anyio==4.0.0rc1
```

Comment:
```suggestion
anyio==3.7.1
```

Reply:
Is this the only change needed that you mentioned on https://github.com/encode/starlette/pull/2246 ?